lib/incoming_message.ts

```diff
@@ -1,6 +1,18 @@
+import { ValueID } from "zwave-js";
+
 interface IncomingCommandStartListening {
   messageId: string;
   command: "start_listening";
 }
 
-export type IncomingMessage = IncomingCommandStartListening;
+interface IncomingCommandNodeSetValue {
+  messageId: string;
+  command: "node.set_value";
```

Comment:
Isn't it better to split this in `instance: 'node'` and `command: 'setValue`  instead?

Reply:
Instances will be `controller|node|driver`

Reply:
or perhaps "target"

Reply:
The downside of doing this is that it's more difficult to map the types.

Now types and the actual commands are keyed by `command`. If we split it, it will be keyed by `(instance, command)`